Cargo.toml

```diff
@@ -59,7 +59,6 @@ fil_actor_verifreg_state = "6"
 fil_actors_shared = "6"
 filecoin-proofs-api = { version = "14.0", default-features = false }
 flume = "0.10"
-frc46_token = "7"
```

Comment:
How did you find this? Do we need a better unused deps story?

Reply:
I want to use `cargo udeps` again.